### PR TITLE
Added Devtron integration

### PR DESCRIPTION
**Pull request by siddhant-khisty**
Thank you for wanting to submit a Pull Request to the Civo Kubernetes Marketplace repository!

If your pull request is to submit a new application to the marketplace, please answer the following questions:

* [x] Have you followed the guidelines in our Contributing document in setting up the application?
* [x] Have you checked that the application your submission is proposing to add does not yet exist on the Marketplace, or as a pull request to be processed?
* [x] Have you tested your application on the Civo managed Kubernetes service? Please include a screenshot.
 
image

* [x] Can you confirm that the software you are submitting licensed for use in a publicly-available Kubernetes marketplace?
* [ ] Have you included the specific version of the application to be installed?

If your pull request concerns an existing Marketplace application, please make sure you have:

* [ ] Notified the Maintainer of the application in this pull request so that they are aware of your proposal.
* [ ] Outlined the changes you are proposing, and the reasons these are required (e.g. stability, compatibility with new versions of Kubernetes implementations, etc).
* [ ] Tested that the application works with the proposed updates applied (including a screenshot).
* [ ] Updated the version number of the app if that has changed.

### CODE REVIEW

**Comment by siddhant-khisty** [comment]
Hey @saiyam1814, I have made the requested changes. Please review them and let me know if any other changes should be made

**Comment by siddhant-khisty** [comment]
Hey @saiyam1814, any updates on this?

File: devtron/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+name: Devtron
+maintainer: "Devtron Team"
```
Comment (saiyam1814) [inline devtron/manifest.yaml:2]: email would be better, in case we need to contact when the app is not working

**Comment by saiyam1814** [comment]
@siddhant-khisty can you add email instead of DevTron team and its good to be merged

**Comment by siddhant-khisty** [comment]
Hey @saiyam1814, I've added an email.

**Comment by saiyam1814** [comment]
LGTM

**Pull request merged**